### DISCUSSION TITLE
Windows Store App - Screensaver Only?

### DISCUSSION THREAD

**Discussion by dawander** [rocksdanister/lively#2248] [Q&A] [answered]
I'm running the store app on Windows 11.  Wallpaper options works great.  However, I'd prefer to just have a screensaver run when the machine is llocked and not have the wallpaper always on.  I can't figure out a way to make this work.  Does anyone know if it's possible?  If yes, any hints are greatly appreciated.

The reason for the request is that the moving wallpaper is distracting and likely using up resources.  I only want to use resources when machine locked and on screensaver.  Not sure if this can be done through the MS Store app.

Thanks in advance.

**Comment by rocksdanister** [comment, marked as answer]
Screensaver only will work in Store app as long as Lively is running in the background, just dont set any wallpaper.

However Screensaver when machine locked is not possible in the current design of Lively: https://github.com/rocksdanister/lively/discussions/2122

**Comment by dawander** [comment]
I understand.  Thank you for answering so quickly.  I assume this is not something on your current radar screen..

**Reply by rocksdanister** [reply]
Yes, I dont think there is enough demand for screensaver only app/plugin that's 200+mb download.
